src/couch_index/test/couch_index_ddoc_updated_tests.erl

```diff
@@ -0,0 +1,134 @@
+% Licensed under the Apache License, Version 2.0 (the "License"); you may not
+% use this file except in compliance with the License. You may obtain a copy of
+% the License at
+%
+%   http://www.apache.org/licenses/LICENSE-2.0
+%
+% Unless required by applicable law or agreed to in writing, software
+% distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+% WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+% License for the specific language governing permissions and limitations under
+% the License.
+
+-module(couch_index_ddoc_updated_tests).
+
+-include_lib("couch/include/couch_eunit.hrl").
+-include_lib("couch/include/couch_db.hrl").
+
+setup() ->
+    DbName = ?tempdb(),
+    fake_mem3(DbName),
+    fake_index(),
+
+    DbNames = [mem3:name(Sh) || Sh <- mem3:local_shards(mem3:dbname(DbName))],
```

Comment:
This is unusual way of doing things. You could use following if you use `test_util:start_couch([mem3, fabric])` in `ddoc_update_test_()`:
```
ok = fabric:create_db(DbName, [?ADMIN_CTX]),
```
However I see a value in doing it in your way. Since you don't have to start mem3 and fabric and tests are more isolated. 
I think your way is better.